hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -58,6 +66,34 @@
      */
     public static final EvictionPolicy DEFAULT_EVICTION_POLICY = EvictionPolicy.LRU;
 
+    /**
+     * Maximum Size Policy
```

Comment:
Originally I didn't want to touch the `EvictionConfig` at all. I just started with it since I moved some JavaDoc from the `NearCacheConfig` over here. I'll put your comments on the TODO. The `NearCachePreloaderConfig` needs an overhaul as well.